app/controllers/hints_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class DiscourseChat::HintsController < ApplicationController
+  def access_by_category
+    hints = Group
+      .joins(:category_groups)
+      .where(category_groups: { category_id: params[:category_id] })
+      .pluck(:name).map! { |name| "@#{name}" }
```

Comment:
why map! and not map here ?

Reply:
`map!` mutates the array while `map` returns a copy. No difference besides that.

If you prefer map, I can change it.